pytrip/vdx.py

```diff
@@ -1210,6 +1210,14 @@ def get_min_max(self):
         self.temp_max = temp_max
         return temp_min, temp_max
 
+    def is_fully_contained(self):
+        min_pos, max_pos = self.get_min_max()
```

Comment:
Replace indexing with plain variables. Use 3 variables for each condition, along X, Y and Z